File: Typescript/Practice/26072022/easy/easy_hidingAmonstCrowd.ts
#!/usr/bin/env npx ts-node

function detectWord(letters: string): string {
  let word:string = '';
  for (let i = 0; i < letters.length; i++) {
    if (letters[i] === letters[i].toLowerCase()) {
      word += letters[i];
    };
  };
  return word;
};

console.log(detectWord("UcUNFYGaFYFYGtNUH"));
console.log(detectWord("bEEFGBuFBRrHgUHlNFYaYr"));
console.log(detectWord("YFemHUFBbezFBYzFBYLleGBYEFGBMENTment"));

File: Typescript/Practice/easy/easy_hidingAmonstCrowd.py
#!/usr/bin/env python

def detect_word(letters):
  word = ''
  for i in range(len(letters)):
    if letters[i].islower():
      word += letters[i]
  return word

print(detect_word("UcUNFYGaFYFYGtNUH"))
print(detect_word("bEEFGBuFBRrHgUHlNFYaYr"))
print(detect_word("YFemHUFBbezFBYzFBYLleGBYEFGBMENTment"))
